github-org-team-creator.go

```diff
@@ -0,0 +1,103 @@
+package main
+
+import (
+	"os"
+	"github.com/urfave/cli"
+	"fmt"
+	"bufio"
+	"log"
+	"github.com/google/go-github/github"
+	"golang.org/x/oauth2"
+	"strconv"
+)
+
+func main() {
+	app := cli.NewApp()
+	app.Name = "boom"
+	app.Usage = "Adds all users in an organisation to a team of your choice"
+	app.ArgsUsage = "[Organisation] [Team Name]"
+	app.UsageText = fmt.Sprintf("github-org-team-creator %s", app.ArgsUsage)
+	app.Action = func(c *cli.Context) error {
+		reader := bufio.NewReader(os.Stdin)
+		fmt.Print("Please enter your Github personal access token: ")
+		accessToken, _ := reader.ReadString('\n')
+		org := c.Args().Get(0)
+		team := c.Args().Get(1)
+		ts := oauth2.StaticTokenSource(
+			&oauth2.Token{AccessToken: accessToken},
+		)
+		tc := oauth2.NewClient(oauth2.NoContext, ts)
+		githubClient := github.NewClient(tc)
+		teamId := getTeamId(org, team, githubClient)
+		if teamId == 0 {
+			println("Creating team...")
```

Comment:
`println` is not guaranteed to stay in the language: https://golang.org/ref/spec#Bootstrapping